### PR TITLE
React file icon

### PR DESCRIPTION
**Pull request by noamkatzir**
Associated Issue: #2690

added react icon with if the project has react in its sources. not sure we need the search according to folder structure of components which can change from project to project, what do you think?

### CODE REVIEW

File: src/components/SourcesTree.js [hunk 1]
```diff
@@ -161,6 +160,9 @@ class SourcesTree extends Component {
     }
 
     if (!nodeHasChildren(item)) {
+      if (this.props.sources.find(v => /react/.test(v))) {
```
Comment (jasonLaster) [inline src/components/SourcesTree.js]: Hmm, I think we also want to check if the item is in a components directory

Comment (jasonLaster) [inline src/components/SourcesTree.js]: We should also add a feature flag `sourceIcons`

**Comment by jasonLaster** [comment]
Thanks @noamkatzir!

File: src/components/SourcesTree.js (hunk 1 above)
Comment (clarkbw) [inline src/components/SourcesTree.js]: Here are the docs on [feature flags](https://github.com/devtools-html/debugger.html/blob/master/docs/feature-flags.md)

**Comment by jasonLaster** [comment]
Thanks for the help. I'm going to close this as it's gotten a bit stale. Feel free to pick it up again

**Pull request closed without merging**